openmc/mesh.py

```diff
@@ -39,7 +40,8 @@ class MeshBase(IDManagerMixin, ABC):
     bounding_box : openmc.BoundingBox
         Axis-aligned bounding box of the mesh as defined by the upper-right and
         lower-left coordinates.
-
+    indices : Iterable of tuple
+        An iterable of mesh indices for each mesh element, e.g. [(1, 1, 1), (2, 1, 1), ...]
```

Comment:
```suggestion
        An iterable of mesh indices for each mesh element, e.g. [(1), (2), ...]
```
should each entry be a single value instead of 3

Reply:
Negative. For structured meshes, indices are a 3-tupe of (i, j, k) values. In unstructured meshes, we use only one value b/c there aren't additional meaningful dimensions of the mesh.

Reply:
Thanks